PhysicsTools/PatAlgos/python/slimming/modifyPrimaryPhysicsObjects_cff.py

```diff
@@ -22,6 +24,7 @@
 
 modifyPrimaryPhysicsObjects = cms.Sequence( electronMVAValueMapProducer *
                                             photonIDValueMapProducer * photonMVAValueMapProducer * 
+                                            electronRegressionValueMapProducer * photonRegressionValueMapProducer *
```

Comment:
these are not in 75X/76X
why needed here?
Isn't the modification in egmGsfElectronIDSequence enough?